src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/ThrowsTaglet.java

```diff
@@ -25,274 +25,635 @@
 import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.ModuleElement;
+import javax.lang.model.element.PackageElement;
+import javax.lang.model.element.QualifiedNameable;
 import javax.lang.model.element.TypeElement;
 import javax.lang.model.type.ExecutableType;
+import javax.lang.model.type.TypeKind;
 import javax.lang.model.type.TypeMirror;
 import javax.lang.model.util.Types;
 
 import com.sun.source.doctree.DocTree;
 import com.sun.source.doctree.ThrowsTree;
 
 import jdk.javadoc.doclet.Taglet.Location;
+import jdk.javadoc.internal.doclets.formats.html.markup.ContentBuilder;
+import jdk.javadoc.internal.doclets.toolkit.BaseConfiguration;
 import jdk.javadoc.internal.doclets.toolkit.Content;
+import jdk.javadoc.internal.doclets.toolkit.Resources;
+import jdk.javadoc.internal.doclets.toolkit.util.CommentHelper;
 import jdk.javadoc.internal.doclets.toolkit.util.DocFinder;
+import jdk.javadoc.internal.doclets.toolkit.util.Utils;
+
+import static java.util.stream.Collectors.joining;
 
 /**
  * A taglet that processes {@link ThrowsTree}, which represents
- * {@code @throws} and {@code @exception} tags.
+ * {@code @throws} and {@code @exception} tags, collectively
+ * referred to as exception tags.
  */
 public class ThrowsTaglet extends BaseTaglet implements InheritableTaglet {
 
+    /*
+     * Relevant bits from JLS
+     * ======================
+     *
+     * 11.1.1 The Kinds of Exceptions
+     *
+     *   Throwable and all its subclasses are, collectively,
+     *   the exception classes.
+     *
+     * 8.4.6 Method Throws
+     *
+     *   Throws:
+     *     throws ExceptionTypeList
+     *
+     *   ExceptionTypeList:
+     *     ExceptionType {, ExceptionType}
+     *
+     *   ExceptionType:
+     *     ClassType
+     *     TypeVariable
+     *
+     *   It is a compile-time error if an ExceptionType mentioned in
+     *   a throws clause is not a subtype (4.10) of Throwable.
+     *
+     *   Type variables are allowed in a throws clause even though
+     *   they are not allowed in a catch clause (14.20).
+     *
+     *   It is permitted but not required to mention unchecked
+     *   exception classes (11.1.1) in a throws clause.
+     *
+     * 8.1.2 Generic Classes and Type Parameters
+     *
+     *   It is a compile-time error if a generic class is a direct
+     *   or indirect subclass of Throwable.
+     *
+     * 8.8.5. Constructor Throws
+     *
+     *   The throws clause for a constructor is identical in structure
+     *   and behavior to the throws clause for a method (8.4.6).
+     *
+     * General Comments
+     * ================
+     *
+     * Map<ThrowsTree, ExecutableElement> associates a doc tree with its holder
+     * element externally. Such maps are ordered, have non-null keys and values.
+     *
+     * Of all language elements only constructors and methods can declare
+     * thrown exceptions and, hence, document them.
+     */
+
```

Comment:
Would it make sense to add a 3rd section about characteristics of `@throws` tags to be taken into account, citing issues like repeatability, subtypes, and inheritance?

Reply:
Not sure. I'm trying to keep code documentation to a minimum, really. Ideally, all a maintainer would need is "Documentation Comment Specification for the Standard Doclet" and code. I know you are now a fan of such a style, but DRY is good.